connectors/sources/tests/test_sharepoint.py

```diff
@@ -139,11 +139,20 @@ async def test_validate_config_when_host_url_is_empty():
 
 
 @pytest.mark.asyncio
-async def test_validate_config_for_ssl_enabled():
+async def test_validate_config_for_ssl_enabled_when_ssl_ca_not_empty_does_not_raise_error():
     """This function test validate_config when ssl is enabled and certificate is missing"""
     # Setup
-    source = create_source(SharepointDataSource)
-    source.ssl_enabled = True
+    source = create_source(SharepointDataSource, ssl_enabled=True, ssl_ca="test")
+
+    # Execute
```

Comment:
```suggestion
```

I think we should remove these `# Setup`, #` Execute` comments, when we see them. They don't add any value IMO.